### DISCUSSION TITLE
Remove 'tereshkova' from names generator

### DISCUSSION THREAD

**Discussion by ehabibov** [moby/moby#43851] [General]
Hello.

Recently I created some containers with random naming and figured out that received such name:
![2022-07-21_14-06-54](https://user-images.githubusercontent.com/20490584/180204001-b9489978-f03d-4df6-b6ca-aed22c0a9558.jpg)

Valentina Tereshkova is not only a first woman in space (50 yrs ago) but is an active Putin's dictatorship regime supporter who was lobbying Putin's ruling prolongation:
* https://en.wikipedia.org/wiki/Valentina_Tereshkova
_During the drafting process for the [2020 amendments to the Constitution of Russia](https://en.wikipedia.org/wiki/2020_amendments_to_the_Constitution_of_Russia), she proposed to lift the term limits for president Putin.[[68]](https://en.wikipedia.org/wiki/Valentina_Tereshkova#cite_note-69)_
* https://reuters.com/article/us-russia-putin-lawmaker-idUSKBN21021H

It is frustrating seeing such names in my engineering space when I did not ask for that. So I'm asking to remove her name from `pkg/namesgenerator/names-generator.go`

It is obvious that bits, semiconductors, data-structures, software architecture is not about politics but it is impossible to endure and shame to see this knowing how much grief Putin's regime has brought to our free world.

I would rather create PR but it did not work out due to repo setting.
<img width="402" alt="image" src="https://user-images.githubusercontent.com/20490584/180207478-f4295e83-ad1b-4e3d-abd0-75e9ab86f65e.png">

Thank you in advance.
:ukraine:

**Comment by ostr03** [comment]
Absolutely agree. This person is no longer a person who is engaged in space and the progress of mankind.

**Comment by thaJeztah** [comment]
Thanks for bringing this up; that's indeed unfortunate. I guess, it's a risk of adding people to the list that are still living; they may have had major achievements, but can become controversial after that (which is one of the reasons why we decided to freeze the list for new additions in https://github.com/moby/moby/pull/43210).

I agree we should try to keep the list non-controversial, and I'll open a pull request to remove this person.

**Comment by thaJeztah** [comment]
Opened https://github.com/moby/moby/pull/44092 (and back ports to the release-branches)

**Comment by Shekelme** [comment]
You can also stop writing these names and surnames with a capital letter, because in theory they deserve criminal prosecution, but not respect in textual form, albeit formal respect.

**Comment by thaJeztah** [comment]
I think we can close this discussion, because https://github.com/moby/moby/pull/44092 was merged
